src/modules/returns/lib/csv-adapter/mapper.js

```diff
@@ -4,8 +4,20 @@ const util = require('util');
 const parseCsv = util.promisify(require('csv-parse'));
 const moment = require('moment');
 const DATE_FORMAT = 'YYYY-MM-DD';
-const GDS_DATE_FORMAT = 'D MMMM YYYY';
-const GDS_MONTH_FORMAT = 'MMMM YYYY';
+// preferred format for dates is D MMMM YYYY, but some applications
+// may output the CSV in another format. This list attempts to deal
+// with the unexpected formats without risking potential
+// crossovers between formats such at DD/MM/YYYY and MM/DD/YYYY
+const GDS_DATE_FORMATS = [
+  'D MMMM YYYY',
+  'DD MMMM YYYY',
+  'D MMM YYYY',
+  'DD MMM YYYY',
+  'D-MMMM-YYYY',
```

Comment:
I think one I have seen from Excel frequently is:
01-Apr-18
Presumably this is:  DD-MMM-YY